relay-metrics/src/aggregation.rs

```diff
@@ -2720,49 +2773,53 @@ mod tests {
     #[test]
     fn test_flush_bucket() {
         relay_test::setup();
+
         let receiver = TestReceiver::default();
+        // Create the new tokio runtime where we start AggregatorService
+        let rt = tokio::runtime::Runtime::new().unwrap();
+        // Run in the old actix system for receiver
```

Comment:
Do we really still need the old runtime for this test?  Can we not convert TestReceiver to tokio as well?

Reply:
We could, but in production we still have `project_cache` actor which uses old system and we still have to test it. 
Once we convert that actor to the new tokio, we can change all of this as well.

Reply:
sure, but these tests are to make sure the aggregation actor works i think, not to make sure that the callers/senders mess something up?